src/docs/ftw-content/hosted-translations/index.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Hosted translations with Asset Delivery API
+slug: hosted-translations
+updated: 2022-05-13
+category: ftw-content
+ingress:
+  This article describes how hosted translations work in Flex Template
+  for Web (FTW).
+published: true
+---
+
+Starting from 2022-05, FTW template translations can be managed both in
+the built-in translation files and in Flex Console. This article
+describes how the FTW template uses the hosted translations and merges
+them with the built-in translations
+
+_**Note:** If you want to implement this feature into your pre-version
+XX.XX FTW template, you can see the necessary modifications in the PRs
+for [ftw-daily](), [ftw-hourly](), and [ftw-product]() todo: links. Read
```

Comment:
I'd use just the FTW-daily. Other PR:s are coming only after FTW-daily has had release (update is taken from release tag)
https://github.com/sharetribe/ftw-daily/pull/1510

Reply:
Fixed!